src/zkp/pisch.rs

```diff
@@ -175,8 +193,11 @@ impl PiSchProof {
         // Verifier samples e in F_q
         let challenge = positive_challenge_from_transcript(&mut local_transcript, input.q)?;
 
+        // Prover creates a response by adding the randomness generated in the first
+        // step to the challenge multiplied by the secret
         let response = &com.randomness_for_commitment + &challenge * secret.x;
```

Comment:
todo: this comment explains _what_ is happening; it is more useful to explain _why_. E.g.
```
// Create a response by masking the secret with the challenge and mask.